src/components/MenuBar/MenuBar.tsx

```diff
@@ -78,7 +89,7 @@ export default function MenuBar() {
       <Toolbar>
         {roomState === 'disconnected' ? (
           <form className={classes.form} onSubmit={handleSubmit}>
-            {!user?.displayName ? (
+            {window.location.search.includes('customIdentity=true') || !user?.displayName ? (
```

Comment:
How many query parameters do we parse on the URL now? I'm wondering if it might be a good time to create a utility function that returns the value of a query param. Example: `const customIdentity = getQueryParam('customIdentity')`. WDYT?

Reply:
Right now we get query parameters here, and when we consume the passcode (when using passcode authentication). 

So for now I think this is fine, but when we implement a settings menu that (perhaps) consumes a query parameter for each setting (useful for testing), then I think implementing your idea would be good.

Reply:
Ok sounds good. Let's keep this at the back of our head. We don't want to keep this kind of pattern used everywhere.